### PR TITLE
Add test coverage measurements

### PR DESCRIPTION
**Pull request by Tomek52**
issue #15

### CODE REVIEW

File: .codecov.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+coverage:
+  range: 80..100
+  round: down
+  precision: 2
+
+  status:
+    project:                   # measuring the overall project coverage
+      default:                 # context, you can create multiple ones with custom titles
+        enabled: yes           # must be yes|true to enable this status
+        target: 80%            # specify the target coverage for each commit status
```
Comment (LordLukin) [inline .codecov.yml]: auto looks like a better option :)

File: CMakeLists.txt [hunk 2]
```diff
@@ -37,6 +37,9 @@ endif()
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+set(CMAKE_CXX_FLAGS "-g -O0 -Wall -fprofile-arcs -ftest-coverage")
```
Comment (LordLukin) [inline CMakeLists.txt]: I'm not 100% sure but you can probably replace `-fprofile-arcs -ftest-coverage` with `--coverage`

https://stackoverflow.com/questions/28840261/gcov-what-is-the-difference-between-coverage-and-ftest-coverage-when-buildi

File: README.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # Eurobusiness
 [![Build Status](https://www.travis-ci.org/LordLukin/Eurobusiness.svg?branch=master)](https://travis-ci.org/LordLukin/Eurobusiness.svg?branch=master) 
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/56d03d8bdb924fb7af54f6ee2134063d)](https://www.codacy.com/app/LordLukin/Eurobusiness?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=LordLukin/Eurobusiness&amp;utm_campaign=Badge_Grade)
+[![codecov](https://codecov.io/gh/Tomek52/Eurobusiness/branch/master/graph/badge.svg)](https://codecov.io/gh/Tomek52/Eurobusiness)
```
Comment (LordLukin) [inline README.md]: `[codecov](https://codecov.io/gh/LordLukin/Eurobusiness)`

File: .codecov.yml (hunk 1 above)
Comment (Tomek52) [reply .codecov.yml]: We used auto option in bowling project and it's not working good, but we can try do it in this project

File: .codecov.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  range: 80..100
+  round: down
+  precision: 2
+
+  status:
+    project:                   # measuring the overall project coverage
+      default:                 # context, you can create multiple ones with custom titles
+        enabled: yes           # must be yes|true to enable this status
+        target: auto           # specify the target coverage for each commit status
+        treshold: 5%           #   option: "auto" (must increase from parent commit or pull request base)
```
Comment (LordLukin) [inline .codecov.yml]: You have threshold (without a type one line below). Anyway for me looks good now, but please set threshold to 1% only.

**Pull request merged**